belt-dwp/src/lib.rs

```diff
@@ -15,9 +15,9 @@
 #![cfg_attr(not(all(feature = "os_rng", feature = "heapless")), doc = "```ignore")]
 //! # fn main() -> Result<(), Box<dyn std::error::Error>> {
 //! use belt_dwp::{
-//!     aead::{Aead, AeadCore, KeyInit},
-//!     BeltDwp, Nonce
+//!     aead::{Aead, AeadCore, KeyInit}, Nonce, BeltBlock
 //! };
+//! type BeltDwp = belt_dwp::BeltDwp::<BeltBlock>;
```

Comment:
In the examples you can just use `BeltDwp` without defining this explicit alias.